_episodes/02-starting-with-data.md

```diff
@@ -126,10 +126,11 @@ time we call a Pandas function.
 
 We will begin by locating and reading our survey data which are in CSV format. CSV stands for
 Comma-Separated Values and is a common way to store formatted data. Other symbols may also be used, so
-you might see tab-separated, colon-separated or space separated files. It is quite easy to replace
-one separator with another, to match your application. The first line in the file often has headers
-to explain what is in each column. CSV (and other separators) make it easy to share data, and can be
-imported and exported from many applications, including Microsoft Excel. For more details on CSV
+you might see tab-separated, colon-separated or space separated files. pandas can work with each of these
+types of separators, as it allows you to specify the appropriate separator for your data.
+The first line in the file often has headers to explain what is in each column.
```

Comment:
```suggestion

```
I suggest we remove the sentence about the header and start a new paragraph for the part about data sharing and further details about CSV files.